tiledb/cloud/array.py

```diff
@@ -563,3 +602,150 @@ def apply(
         result_format_version=result_format_version,
         **kwargs,
     ).get()
+
+
+def exec_multi_array_udf_async(
+    func=None,
+    array_list=None,
+    name=None,
+    layout=None,
+    image_name=None,
+    http_compressor="deflate",
+    include_source_lines=True,
+    task_name=None,
+    result_format=rest_api.models.UDFResultType.NATIVE,
+    result_format_version=None,
+    **kwargs
+):
+    """
+    Apply a user defined function to multiple arrays
+    :param func: user function to run
+    :param array_details: dict from array name (tiledb format) to tuple (ranges, attributes)
+    :param layout: tiledb query layout
+    :param image_name: udf image name to use, useful for testing beta features
+    :param http_compressor: set http compressor for results
+    :param str task_name: optional name to assign the task for logging and audit purposes
+    :param UDFResultType result_format: result serialization format
+    :param str result_format_version: set a format version for cloudpickle or arrow IPC
+    :param kwargs: named arguments to pass to function
+    :return: UDFResult object which is a future containing the results of the UDF
+    >>> import numpy as np
+    >>> from tiledb.cloud import array
+    >>> import tiledb.cloud
+    >>> def median(numpy_ordered_dictionary):
+    ...     return np.median(numpy_ordered_dictionary["a"])
+
+    >>> dense_array = "tiledb://TileDB-Inc/quickstart_dense"
+    >>> sparse_array = "tiledb://TileDB-Inc/quickstart_sparse"
+    >>> array_details = dict([
+    >>>     (dense_array, ([(1, 4), (1, 4)], ["a"])),
```

Comment:
This example should be updated for the new parameter

Reply:
Updated